package.json

```diff
@@ -37,7 +37,8 @@
     "test-storybook": "test-storybook --watchAll",
     "build-storybook": "storybook build",
     "addlicense": "addlicense -c 'Mia srl' -s -ignore src/**/*.ts",
-    "check-license": "addlicense -c 'Mia srl' -check -s -ignore src/**/*.ts"
+    "check-license": "addlicense -c 'Mia srl' -check -s -ignore src/**/*.ts",
+    "update-themes": "node --no-warnings=ExperimentalWarning --loader ts-node/esm src/themes/scripts/updateThemes.ts"
```

Comment:
why the `--no-warnings=ExperimentalWarning`?

Reply:
because there is an experimental warning that is always printed before the script output. Probably there is a better solution to run a script ts, but this is the first one that working for me.